content/en/database_monitoring/setup_oracle/rac.md

```diff
@@ -25,13 +25,35 @@ Database Monitoring provides deep visibility into your Oracle databases by expos
 
 Complete the following steps to enable Database Monitoring with your database:
 
-1. Configure the Agent for each RAC node by following the instructions for [self-hosted Oracle databases][7].
+Configure the Agent for each RAC node by following the instructions for [self-hosted Oracle databases][7].
 
-2. Add the following custom tag to each node:
-    ```yaml
-    tags:
-      - rac_cluster:your_cluster_name
-    ```
+You must configure the Agent for each RAC node, because the Agent collects information from every node separately by querying `V$` views. The Agent doesn't query and `GV$` views to avoid generating interconnect traffic. The collected data from each RAC node is aggregated in the frontend.
```

Comment:
"The Agent doesn't query and `GV$` views ..." - should "and" be "any" here?

Reply:
Also, I'm not really sure how to interpret the following line: "The collected data from each RAC node is aggregated in the frontend."

Why is this significant to me as a user?

Reply:
> "The Agent doesn't query and `GV$` views ..." - should "and" be "any" here?

corrected now.



> Also, I'm not really sure how to interpret the following line: "The collected data from each RAC node is aggregated in the frontend."
> 
> Why is this significant to me as a user?

We want to be able to analyse the total database load. To do that, we have to aggregate the load generated by each node, as we are collecting statistics from each node separately.

Btw., Raymond James people explicitly asked about how we collect data in RAC. They mentioned our competitor aggregates the information in a wrong way, so that the information about each individual node is lost. We offer both perspectives: per node and for the whole clustered database. I noticed they were pleased with the answer.

Reply:
Nice! This will be an awesome differentiator for us :)